app/tests/src/ingestion/test_pdf_stylings.py

```diff
@@ -0,0 +1,62 @@
+from src.ingestion.pdf_elements import Heading
+from src.ingestion.pdf_stylings import Styling, extract_stylings
+
+
+def test_extract_styles():
+    with open("/app/tests/src/util/707.pdf", "rb") as fp:
+        _stylings = extract_stylings(fp)
+
+    assert _stylings == all_expected_stylings
+
+
+all_expected_stylings = [
+    Styling(
+        text="CDC not eligible due to 6 month penalty period",
+        pageno=3,
+        headings=[Heading(title="Disqualifications", level=1, pageno=2)],
+        wider_text="• First occurrence - six month disqualification. The "
+        "closure reason will be CDC not eligible due to 6 month "
+        "penalty period. ",
+        bold=True,
+    ),
+    Styling(
+        text="CDC not eligible due to 12 month penalty period. ",
+        pageno=3,
+        headings=[Heading(title="Disqualifications", level=1, pageno=2)],
+        wider_text="• Second occurrence - twelve month disqualification. The "
+        "closure reason will be CDC not eligible due to 12 month "
+        "penalty period. ",
+        bold=True,
+    ),
+    Styling(
+        text="CDC not eligible due to lifetime penalty. ",
+        pageno=3,
+        headings=[Heading(title="Disqualifications", level=1, pageno=2)],
+        wider_text="• Third occurrence - lifetime disqualification. The "
+        "closure reason will be CDC not eligible due to lifetime "
+        "penalty. ",
+        bold=True,
+    ),
+    Styling(
+        text="CDC penalty period has ended. See BEM 704 for re-enroll-ment " "requirements.",
```

Comment:
I think the formatter put these on the same line:
```suggestion
        text="CDC penalty period has ended. See BEM 704 for re-enroll-ment requirements.",
```